# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/IDBCursor.h
# Commit: 8dc185488801
# Full Hash: 8dc185488801ddb66e681cedb9b52941437561a8
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-11-05 21:49:23
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Send extra records with every ObjectStoreCursorResponse if enabled by pref. r=ttung,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43252
# ==============================================================================

diff -r 9d8b02674d45 -r 8dc185488801 dom/indexedDB/IDBCursor.h
--- a/dom/indexedDB/IDBCursor.h	Tue Nov 05 13:24:31 2019 +0000
+++ b/dom/indexedDB/IDBCursor.h	Tue Nov 05 12:48:36 2019 +0000
@@ -120,6 +120,8 @@
 
   IDBCursorDirection GetDirection() const;
 
+  bool IsContinueCalled() const { return mContinueCalled; }
+
   void GetKey(JSContext* aCx, JS::MutableHandle<JS::Value> aResult,
               ErrorResult& aRv);
 
@@ -159,6 +161,8 @@
     mBackgroundActor = nullptr;
   }
 
+  void InvalidateCachedResponses();
+
   NS_DECL_CYCLE_COLLECTING_ISUPPORTS
   NS_DECL_CYCLE_COLLECTION_SCRIPT_HOLDER_CLASS(IDBCursor)
 